g/grails/grails_ubi8.5.sh

```diff
@@ -0,0 +1,56 @@
+# -----------------------------------------------------------------------------
+#
+# Package		: grails
+# Version		: v5.0.2
+# Source repo	: https://github.com/grails/grails-core.git
+# Tested on	: ubi 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sapna Shukla <Sapna.Shukla@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+HOME_DIR=`pwd`
+PACKAGE_NAME=grails-core
+PACKAGE_VERSION=${1:-v5.0.2}
+PACKAGE_URL=https://github.com/grails/grails-core.git
+mkdir -p /home/tester
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+yum install git unzip wget maven -y
+
+#clone the repository
+git clone $PACKAGE_URL
+
+export WORK_DIR=$HOME_DIR/$PACKAGE_NAME
+cd $WORK_DIR
+git checkout $PACKAGE_VERSION
+
+export HOME=/home/tester
+wget https://services.gradle.org/distributions/gradle-6.9.1-bin.zip -P /tmp && unzip -d /home/tester/gradle -f /tmp/gradle-6.9.1-bin.zip
+export GRADLE_HOME=/home/tester/gradle/gradle-6.9.1/
+rm -rf /tmp/gradle-6.9.1-bin.zip
+export PATH=${GRADLE_HOME}/bin:${PATH}
```

Comment:
We are using `./gradlew` at Line 42. Hence that script should handle setting gradle related setup.
Is there any reason for doing it explicitly ?

Reply:
Yes, re-ran the script after removing the Gradle setup details, working correctly.
Thanks.